src/layouts/GroupRenderer.tsx

```diff
@@ -0,0 +1,20 @@
+import { GroupLayout, OwnPropsOfRenderer } from "@jsonforms/core"
+import { UISchema } from "../ui-schema";
+import { Divider } from "antd"
+import { AntDLayoutRenderer } from "./LayoutRenderer"
+
+export type LayoutRendererProps = OwnPropsOfRenderer & {
+  elements: UISchema[]
+}
+
+export function GroupComponent({ visible, enabled, uischema, ...props }: LayoutRendererProps) {
```

Comment:
Trying to wrap my head around these `layouts`. Why is this called `GroupComponent` but the file is called `GroupRenderer`? I don't love the names "component' or "renderer" TBH. What would you think if we called them both `GroupLayout` and aliased the jsonforms interface?